pkgs/applications/editors/kakoune/plugins/overrides.nix

```
@@ -158,4 +158,21 @@ declare-option -hidden str ansi_filter %{'"$out"'/bin/kak-ansi-filter}
       platforms = platforms.all;
     };
   };
+
+  kakoune-catppuccin = buildKakounePluginFrom2Nix {
+    pname = "kakoune-catppuccin";
+    version = "2024-03-29";
```

Per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming:

> The version attribute must start with a digit e.g., "0.3.1rc2" or "0-unstable-1970-01-01".
>
> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.

Following this, the version ought to be this:

```suggestion
    version = "0-unstable-2024-03-29";
```

---

done